recipes/flask_cors/LICENSE.txt

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2016 Cory Dolphin, Olin College
+
```

Comment:
Looks like the license file is part of the tarball, so please remove this file from here.

Reply:
done, thanks!